### DISCUSSION TITLE
how to continue its training!

### DISCUSSION THREAD

**Discussion by Viper3xzx** [nerfstudio-project/nerfstudio#2500] [Q&A]
im new to nerf, i already install nerfstudio and all its needs on my Linux, i can train with "nerfacto" model,but when i choose instant-ngp,it cant work, im so confused :(
![CLPXS%FC5P 8}O8_D4V7`AN](https://github.com/nerfstudio-project/nerfstudio/assets/133622772/e75f803b-efc5-4ba3-a328-4ed2e48ebd1d)

**Comment by maturk** [comment]
Is seems to be working, at least it doesnt look like it crashed, what does the viewer look like if you open the link?

**Reply by Viper3xzx** [reply]
i can see the pictures, but screen is  indiscernible, it's quite different when i use nerfacto, and i cant know his progress

**Reply by Viper3xzx** [reply]
![T{_7B} 3H2_9${MS8C TAV](https://github.com/nerfstudio-project/nerfstudio/assets/133622772/51eef4bd-b232-4336-af91-24e5ad774733)

**Comment by HillYoungs** [comment]
similar question

**Comment by HillYoungs** [comment]
similar question
